src/views/AdditionalInformation/RelatedRequestsTable.test.tsx

```diff
@@ -0,0 +1,241 @@
+          variables: {
+            systemIntakeID: systemIntake.id
+          }
+        },
+        result: {
+          data: {
+            systemIntake: {
+              __typename: 'SystemIntake',
+              id: systemIntake.id,
+              relatedIntakes: [
+                {
+                  id: '1',
+                  requestName: 'related intake 1',
+                  contractNumbers: ['1', '2'],
+                  decisionState: SystemIntakeDecisionState.NO_DECISION,
+                  submittedAt: new Date()
+                }
+              ],
+              relatedTRBRequests: [
+                {
+                  id: '2',
+                  name: 'related trb 1',
+                  contractNumbers: ['3', '4'],
+                  status: TRBRequestStatus.FOLLOW_UP_REQUESTED,
+                  createdAt: new Date()
+                }
+              ]
+            }
+          }
+        }
+      }
+    ];
+
+    render(
+      <MemoryRouter>
+        <MockedProvider mocks={mocks}>
+          <Provider store={mockStore}>
+            <MessageProvider>
+              <RelatedRequestsTable systemIntakeID={systemIntake.id} />
+            </MessageProvider>
+          </Provider>
+        </MockedProvider>
+      </MemoryRouter>
+    );
+
+    expect(await screen.findByRole('link', { name: 'related intake 1' }));
+    expect(await screen.findByRole('link', { name: 'related trb 1' }));
+  });
+
+  it('should provide non-clickable text if user is not an admin', () => {
+    const mockStore = easiMockStore({
+      groups: ['EASI_P_USER']
+    });
+
+    const mocks = [
+      {
+        request: {
+          query: GetSystemIntakeRelatedRequestsQuery,
+          variables: {
+            systemIntakeID: systemIntake.id
+          }
+        },
+        result: {
+          data: {
+            systemIntake: {
+              __typename: 'SystemIntake',
+              id: systemIntake.id,
+              relatedIntakes: [
+                {
+                  id: '1',
+                  requestName: 'related intake 1',
+                  contractNumbers: ['1', '2'],
+                  decisionState: SystemIntakeDecisionState.NO_DECISION,
+                  submittedAt: new Date()
+                }
+              ],
+              relatedTRBRequests: [
+                {
+                  id: '2',
+                  name: 'related trb 1',
+                  contractNumbers: ['3', '4'],
+                  status: TRBRequestStatus.FOLLOW_UP_REQUESTED,
+                  createdAt: new Date()
+                }
+              ]
+            }
+          }
+        }
+      }
+    ];
+
+    render(
+      <MemoryRouter>
+        <MockedProvider mocks={mocks}>
+          <Provider store={mockStore}>
+            <MessageProvider>
+              <RelatedRequestsTable systemIntakeID={systemIntake.id} />
+            </MessageProvider>
+          </Provider>
+        </MockedProvider>
+      </MemoryRouter>
+    );
+
+    expect(() =>
```

Comment:
https://testing-library.com/docs/guide-disappearance#nottobeinthedocument
Is the preferred approach because it reads like user interaction instead of catching for errors.